drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -229,7 +229,8 @@ def custom_case_notes(...)
     end
 
     def files(**args)
-      resolve_files(**args)
+      # Exclude files that have been uploaded by custom assessments
+      resolve_files(object.files.left_joins(:custom_data_element).where(custom_data_element: { id: nil }), **args)
```

Comment:
This isn't really a problem, and will probably be fine going forward for the `id` column, but generally we write these as:
```suggestion
      resolve_files(object.files.left_joins(:custom_data_element).merge(Hmis::Hud::CustomDataElement.where(id: nil)), **args)
```

That way we aren't confused by the rails-ism where the table name is probably pluralized, but the hash key isn't.  Basically it just makes the model tell you the scope to merge in.